docs/measures.md

```diff
@@ -169,6 +169,9 @@ This command also respects the `--skip-existing` flag.
 This will prevent it from recalculating the measure for any months or weeks which have already been calculated.
 However the final step, which combines output across time periods, will always be run.
 
+This command accepts an `--output-dir` argument, which defines the directory in which the output measure files will be created. 
+If this is configured, the command will also look for the input files in this directory, and so both this and the generate_cohort command in step 2 must be configured identically.
```

Comment:
This is OK as is :slightly_smiling_face: The below is just for polish more than anything.

nit 1: The "generate_cohort" here would be better inside backticks for consistency :sparkles: (That formatting looks to be the convention throughout the measures documentation.)

nit 2: So I don't have much context here. But I did manage to pick up the meaning, so this is OK as is. However, the second sentence might be slightly easier to follow by rewording to something like:

> If you configure the `--output-dir` argument, `generate_measures` expects the *input* cohort files to be in that same directory. To ensure this, include the same `--output-dir` argument in the previous `generate_cohort` step.

It is tricky to phrase the message here cleanly, though :thinking:

(This might also be better suggested under the `generate_cohort` step. That's assuming someone reads top to bottom. Instead of the reader doing a step, and then seeing they should have done that differently. But I don't have too strong opinion an opinion on that.)

Reply:
I'll approve and leave up to @Jongmassey as to if he wants to edit further. I'd suggest the `generate_cohort` fix is a quick one.

Reply:
Yeah, I was struggling with the phrasing. Everything else seems to be in passive-ish voice so have phrased accordingly